src/common/transformations/include/ov_ops/gather_compressed.hpp

```diff
@@ -38,12 +39,19 @@ class GatherCompressed : public ov::op::v8::Gather {
 
     std::shared_ptr<Node> clone_with_new_inputs(const ov::OutputVector& new_args) const override;
 
-    ov::element::Type get_output_type() const { return m_output_type; }
+    ov::element::Type get_output_type() const {
+        return m_output_type;
+    }
+
+    int64_t get_batch_dims() const {
+        return m_batch_dims;
+    }
 
 protected:
     ov::element::Type m_output_type;
+    int64_t m_batch_dims = 1;
```

Comment:
Do we need this? `op::util::GatherBase` already has `m_batch_dims` and `get_batch_dims()`.

Reply:
Done.